### PR TITLE
VACMS-10657 deprecate uninstall process

### LINKED ISSUE #10657: Remove uninstall step and mark uninstall function as deprecated,
## Description
It was determined [in this PR](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/10650) that it is no longer necessary to programatically uninstall a module.  A module's uninstall hook is fired when the module is disabled using config.

## Acceptance Criteria
- [ ] Function is marked as deprecated in va_gov_db
- [ ] Reference to using it is removed from 

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

Closes #10657

## QA steps

- [ ] validate that ide shows the function as depricated
image

- [ ] validate that readme looks good. https://github.com/department-of-veterans-affairs/va.gov-cms/blob/692dae2381f58a117159cb2fd5865c8484a6502f/READMES/workflow.md

image

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: docroot/modules/custom/va_gov_db/va_gov_db.install [hunk 1]
```diff
@@ -24,6 +24,13 @@ use Psr\Log\LogLevel;
  * @return string
  *   A message about what requested modules were uninstalled.
  *
+ * @deprecated in va_gov_db:9.1.1 and is removed from va_gov_db:15.1.1 This is
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_db/va_gov_db.install:27]: I did not want to fully depreciate it which Is why I punted it way down the road to V15 of a module that is not even versioned.

**Comment by swirtSJW** [comment]
Doh!  Stupid phpstan being too good at his job
image

I guess it will have to not be officially deprecated.  :(

**Comment by ndouglas** [comment]
You could also add it to the `phpstan-baseline.neon`.  I think deprecating it is appropriate 🙂

**Review by ndouglas — DISMISSED** [review]

**Comment by swirtSJW** [comment]
image

**Pull request merged**